### ISSUE TITLE
Todo List

### ISSUE THREAD

**Issue by SachinVin** [SachinVin/citra_android#1]
[ ] Fix Settings
[ ] On-screen controls don't rotate properly with the screen
[ ] Fix Bottom Screen
[ ] Fix Audio

**Comment by ShadyGame** [comment]
you still working on this? its amazing

**Comment by shim80** [comment]
Hi. Are you working on this alone?
Or do you work with Citra Team?

**Comment by SachinVin** [comment]
Hey @ShadyGame,
Thanks, I am but not very actively though.
Hey @shim80,
Solo-ing for now mostly because the code quality is a mess.

**Comment by shim80** [comment]
You can't keep to work alone. This is a big project. Why didn't you conctact Citra Team? They know probably more about what a messy code...
I would have liked to help you, but my programming skills are pretty basic (a little C, C ++ and C #) ...

**Comment by TechUtopiaYoutube** [comment]
Hello SachinVIN, my name is Muhammed, I own youtube channel TechUtopia, I am testing your project fully , please give me all details, or upcoming news, upcoming builds so I can share it public on youtube.
I created special playlist for all gameplays with your project. Please watch here and share it on your [pages-](https://www.youtube.com/watch?v=FyqWqeDcw0w&list=PLlOPcx9eGaU7SPKDZ1zauNwfIhmEHSTio)

if I can help , let me know.
You are doing awesome job man. Please invite more people to help you build this up.
You gonna have thousands of emails and requests, you will see. Emulator community for android is HUGE!

**Comment by SachinVin** [comment]
@shim80 This isn't a "big" project.. atleast i never intend it to be so quickly.
Hey @1orangeboy 
Thanks for reporting , can you create a new issue instead of replying here. Also could you provide the logs over there; They are in "[internal_storage]\citra-emu\logs\" 

@TechUtopiaYoutube 
Hello Muhammed,
I see that you were the reason this repo become so popular, thanks for that.
There aren't any plans for for the future, as you can see this is just a hobby I started when citra got the new hardware renderer this year.
Thanks for testing I really appreciate the effort, 
PS. can you try running the game at a 1x resolution scale.(go to "[internal_storage]\citra-emu\config\sdl2-config.ini" search for 'resolution_factor' and make it '1')

**Comment by 1orangeboy** [comment]
@SachinVin我非常想提交日志但是我并不知道如何像你发送这个txt文件，您能告诉我您的邮箱地址么？非常感谢解答，望回复！

**Comment by 1orangeboy** [comment]
@SachinVin I really want to submit the log but I don't know how to send this TXT file like you. Could you please tell me your email address? Thank you very much for your reply!

**Comment by TechUtopiaYoutube** [comment]
> @shim80 This isn't a "big" project.. atleast i never intend it to be so quickly.
> Hey @1orangeboy
> Thanks for reporting , can you create a new issue instead of replying here. Also could you provide the logs over there; They are in "[internal_storage]\citra-emu\logs"
> 
> @TechUtopiaYoutube
> Hello Muhammed,
> I see that you were the reason this repo become so popular, thanks for that.
> There aren't any plans for for the future, as you can see this is just a hobby I started when citra got the new hardware renderer this year.
> Thanks for testing I really appreciate the effort,
> PS. can you try running the game at a 1x resolution scale.(go to "[internal_storage]\citra-emu\config\sdl2-config.ini" search for 'resolution_factor' and make it '1')
> BTW, you can hide the controls(ZL, ZR, C-stick, and Home) in configure controls -> toggle controls

@SachinVin tnx for responding my ninja. Its my honor to promote your project..
Please tell me what else can I do in citra emu config files? Except X1 resolution, can improve the speed more? Any other tips. Please share all tips, i will make special video about that also. I am here. You can respond here or email me in private at techutopiareview@gmail.com

Please respond ASAP.
Can I start my own posts here on your profile project?

**Comment by ShadyGame** [comment]
You should work heavely on this, get some help, set up a Patreon, watch the money pile up

**Comment by SachinVin** [comment]
@1orangeboy its the same issue as @TechUtopiaYoutube reported

**Comment by 1orangeboy** [comment]
@ok, Thank you very much for your reply, I hope the Android 3DS simulator will support the Kylin processor。

**Comment by 1orangeboy** [comment]
@SachinVinok, Thank you very much for your reply, I hope the Android 3DS simulator will support the Kylin processor。

**Comment by Paraskevidis** [comment]
Did the citra team contacted you yet?
They are working on their own port for Android and if you two join teams then who knows what you can achieve together.

Check this if you haven't already

https://www.reddit.com/r/emulation/comments/9lh8aw/citra_now_working_on_android_according_to/

**Comment by M220** [comment]
Two or Maybe three TODOs of this list can be fixed by Updating Source of Dolphin GUI And Citra Because Some of this Problems are Fixed about one or two Months ago
For example : "On-screen controls don't rotate properly with the screen" or "Fix onscreen controls interfering with the touch screen"
Can You Update This Two Source If You Have The Time? Because Working on an updated Source is Much Better Than Working on an outdated source and trying to fix the problems that is already fixed by others :D

**Comment by GuiCardenas** [comment]
@SachinVin Sou um usuário brasileiro do seu emulador  gostei muito do seu  trabalho porém quando abro algum ROM acaba fechando o apk por favor poderia me ajudar. Agradeço.

**Comment by GuiCardenas** [comment]
@SachinVin Sou um usuário brasileiro do seu emulador  gostei muito do seu  trabalho porém quando abro algum ROM acaba fechando o apk por favor poderia me ajudar. Agradeço.

**Comment by SachinVin** [comment]
@Paraskevidis Yes they have contacted me, we're still figuring out how my hack-job code can be fixed

@M220 I update citra's code almost every couple of months ..... Dolphins code I cant because, one i messed up bad when porting. and two a lot of it was basically re-written so it's going to be hard 

@GuiCardenas English please. but from what google translate tells me. Can you provide logs

**Comment by Paraskevidis** [comment]
Great news SachinVin!

Maybe you could abandon the Dolphin UI and make your own unique cool UI like ppsspp, Drastic, dolphin have?

Many people are also sceptical about this port when they see the Dolphin UI with Dolphins settings, we had many fake Dolphin apps before like one gold version of Dolphin in Google Play store lol.
If you can't make a UI then maybe the Citra team can help?

Just tossing ideas

**Comment by siminio7** [comment]
Hey SachinVin, 
great job, YouTube should think about creating an account at https://www.patreon.com to recieve a little support for your effort. Best possibility to geht support from people who cannot contribute with coding

**Comment by GuiCardenas** [comment]
when to launch a new update from emulator

**Comment by Priyam005** [comment]
@TechUtopiaYoutube 1 is the lowest resolution factor and 9 is the highest.. U can change it to 4 to see the magical difference in the resolution!!

**Comment by hkpanchal** [comment]
@SachinVin, can you share your configuration file to get the best fps with decent graphics? Thanks

**Comment by M220** [comment]
@SachinVin Audio and Settings are both fixed in the latest version so You Can Change the description a little bit :p Thanks For Your Hard Work!

**Comment by SachinVin** [comment]
@M220 Frankly OpenGL is way out of my expertise ,I barely managed to put the GLES contraption together. Just hope someone fixes it :)

**Comment by M220** [comment]
@SachinVin
I found some issues with build system, Sound and Performance after your "update" commit,
I fixed my build issue with reverting one of your "update" commit lines ("-DENABLE_CUBEB=0") but the other issues still exists and it's very hard to talk here because everyone subscribed to this issue notifying everytime we talk and they'll get spammed Sooo...
If you have a discord account, can you send your ID to me?
Here's my email (it's a temporary one ) :
rd4qm@vmani.com
If you don't want to send it, it's totally Ok. You don't even know me after all :p but then we probably have to create an issue and talk there.
I'll probably delete this message after some time  because it's not related to this issue.
Thank you for reading my long comment ;)

**Comment by Nishant765** [comment]
@SachinVin are you still working on this emulator? If so can you please tell me when there will be a next update? Thank you so much for the hard work you have done.
